api/go.mod

```diff
@@ -0,0 +1,29 @@
+module github.com/tigera/operator/api
+
+go 1.23.5
+
+require (
+	github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring v0.80.1
```

Comment:
It would be nice to remove this dependency as well

Reply:
This import is used in the `Endpoint` API here https://github.com/tigera/operator/blob/master/api/v1/monitor_types.go#L69 for the `Interval` and `ScrapeTimeout` fields. In order to remove that dependency we would need to change the type of those fields or move the `Endpoint` struct to somewhere else. Moving the struct would require moving the `ServiceMonitor`, ` ExternalPrometheus` & `MonitorSpec` out of the API as well because those structs are all dependent on each other in the `installation_types.go` .

I don't know how/where these definitions are used, so any guidance on this would be much appreciated

Reply:
Sounds like it might be more work than it is worth.

Reply:
Let's ignore for now